terraform/dynamodbwarehouse.tf

```diff
@@ -0,0 +1,48 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name           = "WarehouseDB"
+  billing_mode   = "PROVISIONED"
+  read_capacity  = 20
+  write_capacity = 20
+  hash_key       = "objectID"
+  range_key = "Goods Name"
+
+    attribute {
+        name = "objectID"
```

Comment:
Why not just id? The table name should already tell you which kind of objects are present

Reply:
Ich weiß nicht ganz was du meinst. Wer soll eine id haben? Ich habe doch meine ObjectID als Primary Key

Reply:
So ich weiß dank Irene was du meinst. Ich finde objectID dafür durchaus ok aber hast natürlich dennoch recht das ID allgemeiner wäre